paddle/fluid/framework/ir/mkldnn/cpu_quantize_squash_pass_tester.cc

```diff
@@ -192,6 +224,7 @@ void CountNodeTest(const ProgramDesc& prog, int removed_nodes_num) {
 
   int original_nodes_num = graph->Nodes().size();
   RegisterPass(&graph);
+
```

Comment:
Please keep a consistent approach to inserting empty lines in this and following test methods.
IMO this method can go without empty lines.